packages/driver/src/dom/elements.ts

```diff
@@ -790,8 +802,6 @@ const isScrollable = ($el) => {
   // if we're any other element, we do some css calculations
   // to see that the overflow is correct and the scroll
   // area is larger than the actual height or width
-  const el = $el[0]
-
   const { overflow, overflowY, overflowX } = window.getComputedStyle(el)
```

Comment:
`window.getComputedStyle(el)` throws a nice `Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element' whenever this `el` is undefined, so adding another check here above before this check.